test/ccc/core/cartesian/axis/abstract-cart-axis-panel-spec.js

```diff
@@ -224,24 +224,6 @@ define([
                         });
                     });
 
-                    describe("label width is smaller than height", function() {
```

Comment:
Good question...

With the previous code, no ticks would be hidden, but with the present code, the resulting `tickVisibilityStep` is `2`, which I believe is the correct result.

So, in the end, the relation between labels-max-width and text-height by itself is not enough to determine that no labels should hide. This makes the test's assertion not apply anymore.